src/server/transaction.h

```diff
@@ -157,7 +157,7 @@ class Transaction {
   // State on specific shard.
   enum LocalMask : uint16_t {
     ACTIVE = 1,                 // Set on all active shards.
-    OUT_OF_ORDER = 1 << 2,      // Whether it can run as out of order
+    OUT_OF_ORDER = 1 << 2,      // Whether it can run out of order, meaningless without keylock
```

Comment:
what do you mean "meaningless" ?

Reply:
I don't always clean it to keep the code simple, so only if KEYLOCK_ACQUIRED is actually set, the value should be interpreted. I can change it so KEYLOCK = 0 implies OOO = 0